source/agora/test/NetworkClient.d

```diff
@@ -66,3 +66,49 @@ unittest
 
     assert(0, "Nodes should have same block height");
 }
+
+/// test request timeouts
+unittest
+{
+    import vibe.core.log;
+    import core.thread;
```

Comment:
How dare you

Reply:
what do you mean?

Reply:
I meant the ordering. But actually the `vibe` import is a debug one, right ?

Reply:
I've always ordered them like this:

- local imports first
- dependencies second
- core/std imports after

Reply:
Ah the style guide says
> Imports are organized by packages: agora first, dependencies second, std third, core last.

Ok, I will move the core import last.